src/services/waf-rules-services/create-waf-rules-allowed-service.js

```diff
@@ -0,0 +1,81 @@
+import { AxiosHttpClientAdapter } from '../axios/AxiosHttpClientAdapter'
+import * as Errors from '@/services/axios/errors'
+import { makeWafRulesAllowedBaseUrl } from './make-waf-rules-allowed-base-url'
+
+export const createWafRulesAllowedService = async ({ payload, id }) => {
+  let httpResponse = await AxiosHttpClientAdapter.request({
+    url: `${makeWafRulesAllowedBaseUrl()}/${id}/allowed_rules`,
+    method: 'POST',
+    body: adapt(payload)
+  })
+
+  return parseHttpResponse(httpResponse)
+}
+
+const adapt = (payload) => {
+  return {
+    match_zones: payload.matchZones.map((zone) => {
+      const zoneWithoutMatchoZone = ['path', 'file_name', 'raw_body']
+      if (zoneWithoutMatchoZone.includes(zone.zone)) {
+        zone.matches_on = null
+      }
+      return zone
+    }),
```

Comment:
Sugestão: 
Seria mais apropriado criar uma constante que retorne esse valor. Além disso, a declaração da constante abaixo pode ser movida para fora do bloco do map, uma vez que não há necessidade de redeclará-la a cada iteração.
```suggestion
      const zoneWithoutMatchoZone = ['path', 'file_name', 'raw_body']
```

Reply:
Eu não entendi muito bem o que deve ser melhorado